docs/sources/reference/commandline/cli.md

```diff
@@ -1396,22 +1396,6 @@ just a specific mapping:
     $ sudo docker port test 7890
     0.0.0.0:4321
 
-## pause
-
-    Usage: docker pause CONTAINER
-
-    Pause all processes within a container
-
-The `docker pause` command uses the cgroups freezer to suspend all processes in
-a container.  Traditionally when suspending a process the `SIGSTOP` signal is
-used, which is observable by the process being suspended. With the cgroups freezer
-the process is unaware, and unable to capture, that it is being suspended,
-and subsequently resumed.
-
-See the
-[cgroups freezer documentation](https://www.kernel.org/doc/Documentation/cgroups/freezer-subsystem.txt)
-for further details.
-
```

Comment:
I don't know it was there. I remove it because there was another "pause" on lines 1362-1375